src/test/eo/org/eolang/math/number-tests.eo

```diff
@@ -1083,11 +1114,11 @@
       number 0
     $.equal-to negative-infinity
 
-[] > ln-of-int-one-is-int-zero
+[] > ln-of-int-one-is-float-zero
   assert-that > @
     ln.
       number 1
-    $.equal-to 0
+    $.equal-to 0.0
 
```

Comment:
@levBagryansky maybe we should do a test with a return 'nop' for negative values. But it's up to you.